spec/ddtrace/benchmark/microbenchmark_spec.rb

```diff
@@ -87,9 +87,11 @@ def reset_buffer
 
       let(:steps) { [1, 10, 100] }
 
-      let(:tracer) { Datadog::Tracer.new }
+      let(:tracer) { Datadog::Tracer.new(writer: writer) }
       after { tracer.shutdown! }
 
+      let(:writer) { Datadog::Writer.new(buffer_size: 1000, flush_interval: 0) }
```

Comment:
This ensures that spans are actually being consumed quickly by the writer, instead of being mostly dropped by the buffer due to the 1000 span default limit.

Our benchmarks aim to simulate a realistic user scenario, but as quickly as possible. Having the worker flush spans more frequently helps us accomplish that goal.